packages/glimmer-runtime/lib/compiled/opcodes/vm.ts

```diff
@@ -306,6 +313,77 @@ export class EvaluateOpcode extends Opcode {
   }
 }
 
+export class EvaluatePartialOpcode extends Opcode {
+  public type = "evaluate-partial";
+  public compiler: CompileInto & SymbolLookup;
+  public name: CompiledExpression<any>;
+
+  constructor({ name, compiler }: { compiler: CompileInto & SymbolLookup, name: CompiledExpression<any> }) {
+    super();
+    this.name = name;
```

Comment:
You shouldn't need to take the name – per your compilation, the name should already be evaluated and sitting on the args register at this point